pbs_light.go

```diff
@@ -739,6 +739,11 @@ func init() {
 	viper.SetDefault("adapters.conversant.endpoint", "http://media.msg.dotomi.com/s2s/header/24")
 	viper.SetDefault("adapters.conversant.usersync_url", "http://prebid-match.dotomi.com/prebid/match?rurl=")
 	viper.SetDefault("host_cookie.ttl_days", 90)
+
+	// Set environment variable support:
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
```

Comment:
Subjective bikeshedding here, feel free to ignore:

Would double underscores be more clear? E.g.: `PBS_HOST_COOKIE__TTL_DAYS` maps to `host_cookie.ttl_days`.

Reply:
The example from the viper test cases has one underscore. I am a little wary of double underscores, eyeballing the string you can confuse single and double underscores if you don't examine it carefully. Are there places where people use the double underscore convention?

Reply:
Single underscores makes sense and it looks like viper knows when to replace with a period and when to leave the underscore alone. Please ignore my comment, LGTM.